src/nvim/generators/gen_api_dispatch.lua

```diff
@@ -8,9 +8,11 @@ if arg[1] == '--help' then
   print('      3: functions metadata output file (funcs_metadata.generated.h)')
   print('      4: API metadata output file (api_metadata.mpack)')
   print('      5: lua C bindings output file (msgpack_lua_c_bindings.generated.c)')
+  print('      6: api docs metadata output file (api_doc_metadata.generated.h)')
+  print('      7: api docs msgpack file path (api.mpack)')
   print('      rest: C files where API functions are defined')
 end
-assert(#arg >= 4)
+assert(#arg >= 5)
```

Comment:
Needs to be `#arg >= 7`?

Reply:
Turn it to 7 and see what happens (it seems reasonable, yes)